src/CMakeLists.txt

```diff
@@ -143,13 +143,18 @@ target_include_directories(ObjectStore
        $<BUILD_INTERFACE:${RealmObjectStore_SOURCE_DIR}/src>
        $<INSTALL_INTERFACE:include> 
 )
+target_include_directories(ObjectStore
+    PRIVATE
+        ${RealmObjectStore_SOURCE_DIR}/external/pegtl
```

Comment:
It is now pulled by core, right?

Reply:
That is correct - at some point we’ll rebase the CMake branches and remove the parser from Object Store.